src/app/(main)/EventPage.tsx

```diff
@@ -110,7 +126,11 @@ export const EventPage = ({ eventId }: { eventId: string }) => {
             <OutputLink label="Map" value={activity.mapId} href={`https://sartopo.com/m/${activity.mapId}`} />
           </Box>
           <Box>
-            <OutputText label="Status" value={isActivityActive ? 'In Progress' : 'Complete'} />
+            <OutputText label="Mission Status" value={getActivityStatus(activity)} />

```

Comment:
I'm concerned about overloading the table of mission details such that it becomes difficult to glance at. Perhaps it's worth putting the various numbers of responders in their own table or section for readability?

Reply:
Agreed at some point I think we'll want to put it in a sections and maybe make the entire sections collapsible.